crates/nu-cli/src/completions/completion_common.rs

```diff
@@ -335,15 +335,15 @@ pub fn sort_completions<T>(
         } else {
             matcher = matcher.ignore_case();
         };
-        items.sort_by(|a, b| {
+        items.sort_unstable_by(|a, b| {
             let a_str = get_value(a);
             let b_str = get_value(b);
             let a_score = matcher.fuzzy_match(a_str, prefix).unwrap_or_default();
             let b_score = matcher.fuzzy_match(b_str, prefix).unwrap_or_default();
             b_score.cmp(&a_score).then(a_str.cmp(b_str))
         });
     } else {
-        items.sort_by(|a, b| get_value(a).cmp(get_value(b)));
+        items.sort_unstable_by(|a, b| get_value(a).cmp(get_value(b)));
```

Comment:
Only really worth arguing with good benchmarks but in the case where it is just the value, if there may be runs of presorted data coming from the upstream source, where the [mergesort like components of `sort_by`](https://doc.rust-lang.org/std/primitive.slice.html#current-implementation-9) may have [some benefits](https://doc.rust-lang.org/std/primitive.slice.html#current-implementation-3).